src/main/flight/pid.c

```diff
@@ -145,9 +145,9 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .throttle_boost_cutoff = 15,
         .iterm_rotation = false,
         .smart_feedforward = false,
-        .iterm_relax = ITERM_RELAX_OFF,
-        .iterm_relax_cutoff_low = 3,
-        .iterm_relax_cutoff_high = 30,
+        .iterm_relax = ITERM_RELAX_RPY,
```

Comment:
Happy to add this to 3.4, but the default will have to be reverted to `ITERM_RELAX_OFF`.